README.md

```diff
@@ -1,2 +1,160 @@
-# RoR-Budget-app
-The Ruby on Rails capstone project (remember what they are?) is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.  
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 RoR-Budget-app <a name="about-project"></a>
+
+> The Ruby on Rails capstone project (remember what they are?) is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what. 
+
+## Built With
+
+![](https://img.shields.io/badge/-Ruby-red)
+![](https://img.shields.io/badge/-VSCode-blue)
+![](https://img.shields.io/badge/-Rspec-red)
+
+- Visual Studio Code
+- Ruby
+- Rspec
+- Rails
+
+### Key Features <a name="key-features"></a>
+
+<!-- > Describe between 1-3 key features of the application. -->
+
+- Splash screen
+- Sign up and log in pages
+- Home page (categories page)
+- Transactions page
+- "Add a new category" page
+- "Add a new transaction" page
+- Testing requirements
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
- [ ] Your live demo link should be pointing to the application deployed using Render, Netlify, or github pages. Now it is pointing to your GitHub page. Kindly work on it and add the correct link.